response.go

```diff
@@ -174,11 +174,11 @@ func unpackResponseBody(buf []byte, resp interface{}) error {
 			if err != nil {
 				return perrors.WithStack(err)
 			}
-			atta, ok := attachments.(map[string]string)
-			if ok {
+			if v, ok := attachments.(map[interface{}]interface{}); ok {
```

Comment:
should it also check whether its type is map[string]string, which may be supported in the feature?

Reply:
When `hessian2` support to convert `map[string]string` in the future, `atta, ok := attachments.(map[string]string)` will may  be used.

Reply:
ok